src/modules/src/commander.c

```diff
@@ -154,30 +143,3 @@ int commanderGetActivePriority(void)
 
   return priority;
 }
-
-/**
- *
- * The high level commander handles the setpoints from within the firmware
- * based on a predefined trajectory. This was merged as part of the
- * [Crazyswarm](%https://crazyswarm.readthedocs.io/en/latest/) project of the
- * [USC ACT lab](%https://act.usc.edu/) (see this
- * [blogpost](%https://www.bitcraze.io/2018/02/merging-crazyswarm-functionality-into-the-official-crazyflie-firmware/)).
- * The high-level commander uses a planner to generate smooth trajectories
- * based on actions like ‘take off’, ‘go to’ or ‘land’ with 7th order
- * polynomials. The planner generates a group of setpoints, which will be
- * handled by the High level commander and send one by one to the commander
- * framework.
- *
- * It is also possible to upload your own custom trajectory to the memory of
- * the Crazyflie, which you can try out with the script
- * `examples/autonomous_sequence_high_level of.py` in the Crazyflie python
- * library repository.
- */
-PARAM_GROUP_START(commander)
-
-/**
- *  @brief Enable high level commander
- */
-PARAM_ADD_CORE(PARAM_UINT8, enHighLevel, &enableHighLevel)
-
-PARAM_GROUP_STOP(commander)
```

Comment:
Removing this would break a lot of code out there, right? Including some examples

Reply:
yes indeed, in the cflib, cfclient and also some demo applications

Reply:
Sorry about that! In Crazyswarm missing parameters only cause a warning. I didn't realize it could actually break code.

I will restore the old param and comment. All `crazyflie-firmware` maintainers should be able to push to this branch, so please edit the docstring as you see fit.

Reply:
restored in 4109e90